### PR TITLE
Create ProductList component (empty)

### LINKED ISSUE #2: Create product list page
## Value Statement

As a user, I want to view products, so that I can pick the ones I want in my routine

## Description

Screenshot 2021-04-26 at 12 04 02

## Acceptance Criteria

* Products need to be displayed with name, image, details and "add" option

## Tasks

- [x] Create feature branch
- [x] Create product card component
- [x] Create filter component
- [x] Render product data in component
- [x] Styling product card component and product list page

## Complexity
medium

### PR DESCRIPTION
**Pull request by FriederikeM**
* Mock up product card component
* Set up filter forms
* Styling page and card component

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@headlessui/react": "^1.1.0",
```
Comment (hugh-burgess) [inline package.json:6]: gr8

File: src/color.css [hunk 2]
```diff
@@ -1,10 +1,12 @@
 :root {
   --color-one: #fff7fc;
   --color-two: #691643;
+  --color-two-opaque: #691644a8;
```
Comment (hugh-burgess) [inline src/color.css:5]: wow so many colours defined here.. do that! ;)

File: src/components/CalendarButton.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.calendar {
+  font-size: 50px;
+  border: 1px solid var(--color-two);
+  border-radius: 5px;
+  padding: 5px 5px 5px 7px;
+  background-color: #e4bcbc7a;
```
Comment (hugh-burgess) [inline src/components/CalendarButton.css]: why are there so many digits after the hashtag here? I smell cheating!

File: src/components/CalendarButton.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { ImCalendar } from "react-icons/im";
+import { NavLink } from "react-router-dom";
+import "./CalendarButton.css";
+
+export default function CalendarButton() {
+  return (
+    <div className="IconButton">
+      <NavLink to="/weekly-routine" className="weekly-routine-link">
```
Comment (hugh-burgess) [inline src/components/CalendarButton.js:8]: you can just use Link instead of NavLink here buddy

File: src/components/NameFilter.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  width: 200px;
+}
+
+.name-search-button {
+  border-radius: 50%;
+  background-color: var(--color-one);
+  border: 2px solid var(--color-two);
+  color: var(--color-two);
+  backdrop-filter: blur(8px);
+  padding: 7px 9px 5px;
```
Comment (hugh-burgess) [inline src/components/NameFilter.css:23]: 3 values? heh?

File: src/components/ProductCard.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import "./ProductCard.css";
+import Niacinamide from "../images/niacinamide-bottle.png";
+
+export default function ProductCard() {
+  return (
+    <div className="ProductCard">
+      <h5 className="ProductCard__headline">Niacinamide 10% + Zinc 1%</h5>
+      <img src={Niacinamide} alt="product bottle" className="bottle-image" />
```
Comment (hugh-burgess) [inline src/components/ProductCard.js]: the `alt` is for screen readers so you might wanna be more specific for blind customers wanting Niacinamide..

File: src/components/ProductList.css [hunk 7]
```diff
@@ -0,0 +1,25 @@
+.ProductList {
+  display: grid;
+  grid-template-rows: [header] 100px [main] calc(100vh - 100px);
```
Comment (hugh-burgess) [inline src/components/ProductList.css:3]: what is this wizardry???

**Review by hugh-burgess — CHANGES_REQUESTED** [review]
LGLGLTMTMGT. some notes

**Review by soerendolling — APPROVED** [review]
lgtm

**Pull request merged**